resources/services/cloudwatchlogs/gen.hcl

```diff
@@ -0,0 +1,41 @@
+resource "aws" "cloudwatchlogs" "log_groups" {
+  path = "github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types.LogGroup"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["logs"]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSRegion"
+    }
+  }
+
+  userDefinedColumn "tags" {
+    type              = "json"
```

Comment:
Add `description` here, something like `The tags for the log group.`
(https://github.com/aws/aws-sdk-go-v2/blob/205e0e89f662df907b2724676467a77384f15162/service/cloudwatchlogs/api_op_ListTagsLogGroup.go#L41)

Reply:
Got it, thank you!